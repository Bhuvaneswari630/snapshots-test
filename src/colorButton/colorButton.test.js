import ColorButton from './colorButton'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<ColorButton/>).toJSON()
  expect(tree).toMatchSnapshot()
})
