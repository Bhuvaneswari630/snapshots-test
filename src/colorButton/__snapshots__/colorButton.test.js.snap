// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div>
  <h1>
    Testing Playground
  </h1>
  <p>
    Button is 
    disabled
  </p>
  <button
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "blue",
      }
    }
  >
    Change button color to 
    green
  </button>
  <input
    defaultChecked={false}
    onChange={[Function]}
    type="checkbox"
  />
</div>
`;
